modules/neo4j/neo4j.go

```diff
@@ -0,0 +1,84 @@
+package neo4j
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"net/http"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+const defaultImageName = "neo4j"
+const defaultTag = "4.4"
+const defaultBoltPort = "7687"
+const defaultHttpPort = "7474"
+const defaultHttpsPort = "7473"
+
+// Neo4jContainer represents the Neo4j container type used in the module
+type Neo4jContainer struct {
+	testcontainers.Container
+}
+
+func (c Neo4jContainer) BoltUrl(ctx context.Context) (string, error) {
+	host, err := c.Host(ctx)
+	if err != nil {
+		return "", err
+	}
+	containerPort, err := nat.NewPort("tcp", defaultBoltPort)
+	if err != nil {
+		return "", err
+	}
+	mappedPort, err := c.MappedPort(ctx, containerPort)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("neo4j://%s:%d", host, mappedPort.Int()), nil
+}
+
+// StartContainer creates an instance of the Neo4j container type
+func StartContainer(ctx context.Context, options ...Option) (*Neo4jContainer, error) {
+	settings := config{
+		imageCoordinates: fmt.Sprintf("%s:%s", defaultImageName, defaultTag),
```

Comment:
Could we prepend "docker.io" as registry in a explicit manner? That will be inclusive with other container runtimes, not only for Docker.

Reply:
Fixed in 2457831302d3466bd6b64041131b043dd95a770e